### ISSUE TITLE
recap is "touching" rs-sysmon package files - should recap obsolete rs-sysmon

### ISSUE THREAD

**Issue by tonyskapunk** [rackerlabs/recap#59]
Noticed the [%post](https://github.com/rackerlabs/recap/blob/master/util/packaging/rpm/recap.spec#L50-L71) script is doing some changes if `rs-sysmon` files are found in the system where `recap` is being installed, based on those I submitted #58 but since there is no interaction between `recap` and `rs-sysmon` RPMs I think this is a good opportunity to define whether or not `recap` should `Obsoletes: rs-sysmon` ?

IMO seems like this is a good use case of the use of `Obsoletes` as `recap` is a [rename](https://github.com/rackerlabs/recap/wiki/All-about-recap-and-historical-information) of `rs-sysmon`

Comments?

**Comment by man-chung** [comment]
Yes, we should Obsolete and remove the %post script. +1 from me

**Comment by tonyskapunk** [comment]
PR #60 proposes `Conflicts` I suggest `Obsoletes` as mentioned in here https://github.com/rackerlabs/recap/pull/60#issuecomment-222166268 any comments, ideas or suggestions?

**Comment by carlwgeorge** [comment]
[Best practice](https://fedoraproject.org/wiki/Packaging:Guidelines#Renaming.2FReplacing_Existing_Packages) is to put an upper limit on the obsoletes, just in case you ever need to revert it.  The last version of rs-sysmon I can find was 0.9.5-1.rs, so we should use:

```
Obsoletes: rs-sysmon < 0.9.5-2
```

Also, if we feel that recap is a "compatible enough replacement" for rs-sysmon, we should have it provide the rs-sysmon name.  That would allow users to continue to run `yum install rs-sysmon`, which would resolve to install recap instead.

```
Provides: rs-sysmon = %{version}-%{release}
```

**Comment by bhgraham** [comment]
Definitely, that was always the goal to be the full replacement.  I am all for adding the provides and obsoletes.  

And if we are saying it is stable to be the replacement, we should version up to 1.0 as well.

**Comment by tonyskapunk** [comment]
@carlwgeorge  @bhgraham thanks for the input, I'll make sure `Obsoletes` and `Provides` are included in #60.

Regarding bumping to `1.0` I'd wait as there are still some PR and issues reported, trying to catch up on those.

**Comment by tonyskapunk** [comment]
Fix on #89
